### ISSUE TITLE
[BUG]: Decending X-Axis changes axis titles and data, but not tooltips/hover

### ISSUE THREAD

**Issue by jberry** [jneilliii/OctoPrint-BedLevelVisualizer#490]
**Describe the bug**
When you select "Descending x-axis", the data and axis titles rotate, but when you hover-over the data, it's showing the non-flipped-data for rotation

**Expected behavior**
I expect the tooltips/hover to be correct

**Screenshots**
![xaxis1](https://user-images.githubusercontent.com/6216867/146043401-3dae8cbb-5f4f-4845-a695-2bcab053e0f0.png)
![xaxis2](https://user-images.githubusercontent.com/6216867/146043441-fdc18967-e5ed-49ab-b9c2-406627ec55ad.png)

**Firmware and Version**
Bed Visualizer Settings version 1.1.0

**Comment by jneilliii** [comment]
Thank you for your report. When I get some time I'll investigate this further. I've never really been happy with how this table works (was contributed by someone else).

**Comment by jberry** [comment]
Yeah, I'm not overly fond of it, especially when you start rotating the graphic and the table is static, and you start adjusting things wrong (totally user fault: mine).  But it'd be nice if the graphic at least loaded in the same orientation as the table's data layout (i.e. front edge in my graphic should load as 315:340, not 0:0, when the table is flipped via checkboxes/etc.)

I haven't checked out the project yet to poke around myself, but if I get time I'll see if I can as well

**Comment by jneilliii** [comment]
rather than actually fixing this I'm just going to take the hover tooltip out completely. It just doesn't seem to be beneficial and taking it out should actually improve processing time.

**Comment by jberry** [comment]
This did not resolve the data issue of the right chart, the data is still flipped. As you can see in my screenshot below, mousing over the blue 50 shows the data for the red 3:
![image](https://user-images.githubusercontent.com/6216867/147893567-3f83f6ed-27e8-4475-a97e-31eeecf44ddf.png)
And here's the opposite, mousing over the red 3 shows the data for the blue 50:
![image](https://user-images.githubusercontent.com/6216867/147893600-25666d31-e9ff-43d5-a120-6432b7fcd0cd.png)

**Comment by jneilliii** [comment]
Sorry I wasn't paying attention. I thought the issue that the arrow on the mouse pop-over was incorrect, not the actual calculated turns.
